proto/buf/alpha/registry/v1alpha1/repository.proto

```diff
@@ -46,6 +47,23 @@ message Repository {
   bool deprecated = 8;
   // deprecation_message is the message shown if the repository is deprecated.
   string deprecation_message = 9;
+  // owner_name is the name of the owner of the repository,
+  // either a username or organization name.
+  string owner_name = 10;
+  // description is the user configurable description of the repository.
+  string description = 11;
+  // url is the user configurable URL in the description of the repository,
+  // always included the scheme and will not have a #fragment suffix.
+  string url = 12;
+  // license is the user configurable license of the repository, which should be one of
+  // the identifier defined in https://spdx.org/licenses
+  string license = 13;
+  // license url is the user configurable url to the license text.
+  string license_url = 14;
+  // verification status of the owner of the repository, configurable by server admin.
+  VerificationStatus verification_status = 15;
+  // last commit time in main branch of the repository.
+  google.protobuf.Timestamp last_commit_time = 16;
```

Comment:
These two properties don't feel like they belong on the registry repository object, why are we adding them here rather than expecting the user to get them through the owner/listing commits?

Reply:
make sense to me, but I assume frontend folks would like to prevent 2 more calls just for rendering the repository page? (cc @timostamm @paul-sachs ), how do you think if we add them to `GetRepositoryResponse` and `GetRepositoryByFullNameResponse`, just like repository counts?

Reply:
For the frontend, a single endpoint that provides all necessary information to render the page would be desirable. It is much less overhead on the wire, and less code to reason about when working on the actual UI.  
This becomes a more serious limitation in list views: Without last commit time and verification status embedded in the message, it is simply not feasible to show the information in lists. Without an owner name, it is not possible to link to the repository (because we would want canonical URLs with names).

It makes perfect sense to me to understand Repository as the normalized representation of the entity. The verification status obviously belongs on the object representing the owner from that POV, and adding it to both entities adds ambiguity. 

This looks like two opposing goals to me, and I'm not sure there is a simple answer. Compared to popular public APIs, the toll does seem quite high for the FE.

Reply:
@robbertvanginkel and I have discussed on this offline [previously](https://github.com/bufbuild/buf/pull/1368#discussion_r968887617), and we think we might end up needing something frontend specific. But that will need a separated discussion and I think we can move this forward for now